va/va.h

```diff
@@ -1036,6 +1036,16 @@ typedef enum {
      * The value returned uses the VAConfigAttribValEncPerBlockControl type.
      */
     VAConfigAttribEncPerBlockControl    = 55,
+    /**
+     * \brief Maximum number of tile rows and cols. Read-only.
+     *
+     * This attribute determines the maximum number of tile
+     * rows and columns supported for encoding with tile support.
+     *
+     * The bottom 16 bits specify the maximum number of rows
```

Comment:
Is there an actual need to complicate working with this attribute and/or this aligns to something else in VAAPI? I mean, maybe separate `VAConfigAttribEncMaxTileRows` and `VAConfigAttribEncMaxTileCols` will be easier for user?

Reply:
Thanks for taking a look @dvrogozh . This pattern is used on other caps like `VAConfigAttribEncMaxRefFrames`, and then vainfo prints both values by parsing the low/high bit regions. I'm open to splitting it into `VAConfigAttribEncMaxTileRows` and `VAConfigAttribEncMaxTileCols` is that's the consensus, but I think max tile/cols should really be a single cap reported by driver with two values.

```
    /**
     * \brief Maximum number of reference frames. Read-only.
     *
     * This attribute determines the maximum number of reference
     * frames supported for encoding.
     *
     * Note: for H.264 encoding, the value represents the maximum number
     * of reference frames for both the reference picture list 0 (bottom
     * 16 bits) and the reference picture list 1 (top 16 bits).
     */
    VAConfigAttribEncMaxRefFrames       = 13,
```

Reply:
@dvrogozh I pushed the change to split them into two separate queries. Taking deeper look I found other existing caps that are queried separately and I agree with you it may be easier for the user to just get the value without having to parse the low/high bits. 

ps. Also updated the vainfo PR: https://github.com/intel/libva-utils/pull/344

Reply:
> ps. Also updated the vainfo PR: https://github.com/intel/libva-utils/pull/344

Was about to ask. Thanks for raising up.